script/ci.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+ROOT_DIR="${DIR}/.."
+cd "$ROOT_DIR"
+
+yarn install
+yarn type-check
+yarn lint
+yarn bundle
+
+cp package.json dist
+cp riff-raff.yaml dist
```

Comment:
We shouldn't need to move the `riff-raff.yaml` file around, as we're configuring `guardian/actions-riff-raff@v2` to find it in it's original location.